File: framework/languages/java/samples/java/Bingo/Server/Play/src/main/java/systems/zlink/samples/bingo/server/play/application/roomallocation/BingoRoomAllocation.java
package systems.zlink.samples.bingo.server.play.application.roomallocation;

import systems.zlink.samples.bingo.server.play.domain.bingo.BingoRoomModels;

public record BingoRoomAllocation(
    String roomId,
    BingoRoomModels.BingoRoomSettings settings) {
}

File: framework/languages/java/samples/kotlin/Bingo/Server/Play/src/main/kotlin/systems/zlink/samples/kotlin/bingo/server/play/application/roomallocation/BingoRoomAllocation.kt
package systems.zlink.samples.kotlin.bingo.server.play.application.roomallocation

import systems.zlink.samples.kotlin.bingo.server.play.domain.bingo.BingoRoomSettings

data class BingoRoomAllocation(
    val roomId: String,
    val settings: BingoRoomSettings,
)
